docs/standard/native-interop/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Native interoperability
+description: Learn how to interface with native components in .NET.
+author: blackdwarf
+ms.author: ronpet
+ms.date: 06/20/2016
+ms.technology: dotnet-standard
+---
+# Native Interoperability
+
+In this section, we will dive a little bit deeper into all three ways of doing "native interoperability" that are available using .NET.
+
+There are a few of reasons why you would want to call into native code:
```

Comment:
Microsoft's style also asks for the use of contractions: https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#project-friendliness

you'd, aren't, etc.